apps/ledger-live-desktop/src/renderer/actions/accounts.ts

```diff
@@ -54,7 +58,11 @@ export const updateAccount: UpdateAccount = payload => ({
   },
 });
 
-export const toggleStarAction = (id: string, parentId: ?string) => {
+type ToggleAction = {
+  type: string;
+  payload: { updater: (account: Account) => any; accountId: string | undefined };
```

Comment:
I think that you can use the `?` instead of `undefined` here
```suggestion
  payload: { updater: (account: Account) => any; accountId?: string };
```